### ISSUE TITLE
How to setting suggestion list when typing exactly?

### ISSUE THREAD

**Issue by MrHOY** [atom/autocomplete-plus#934]
Ex: 
![atom](https://user-images.githubusercontent.com/20653706/33466088-44279f02-d67e-11e7-9b12-727d2c79a51f.png)

When I type 'imts', imts should on top (imts is my snippet). How can I do that ?

**Comment by leroix** [comment]
In this particular case, `imts` is coming from the snippets autocomplete provider which has a lower priority than the provider from which `implements` is coming from. The green arrow means "snippet".

**Comment by siestamark** [comment]
I agree. Autocomplete+ would be so much more useful if its first choice would be exactly what is typed (if it shows up in the options list.) Please work on sorting as such.
